Testscripts/Linux/SetupRDMA.sh

```diff
@@ -187,7 +187,8 @@ function Main() {
 		suse*|sles*)
 			# install required packages
 			LogMsg "Starting RDMA setup for SUSE"
-			req_pkg="bzip expect glibc-32bit glibc-devel libgcc_s1 libgcc_s1-32bit libpciaccess-devel gcc-c++ gcc-fortran rdma-core libibverbs-devel librdmacm1 libibverbs-utils bison flex numactl"
```

Comment:
You removed one 'numactl' recently added. Please sync your branch before development starts